source/standards/threat-modelling.html.md.erb

```diff
@@ -0,0 +1,236 @@
+mitigation outweighs the maximum cost of a threat.
+
+### 4. Did we do a good enough job?
+
+After completing the threat modelling activity sessions, you should set some
+time to review the threat modelling outputs regularly. This is to see whether
+you are improving the security and taking the mitigations forward; it’s also
+important to consider any new threats.
+
+A recommendation is to perform this review within your service team every two
+months; 90 minutes is usually sufficient, but the time you need may vary
+depending on the scope of your threat modelling.
+
+You should consider running the threat modelling sessions every year or when
+there are significant architectural changes planned.
+
+## Why's it necessary?
+
+Adam Shostack has a great short video on this: [Why Threat Model?].
+
+Adam explains that we threat model to anticipate problems when it's inexpensive
+to deal with them.
+
+How do you know what a problem is? Information and computer security is
+often broken up into three main pillars, known as the CIA triad:
+
+### Confidentiality
+- Information accessible to authorised people
+
+### Integrity
+- Information that's correct
+
+### Availability
+- Information available when needed
+
+Abuse or neglect of one or more of these pillars could lead to a security
+incident.
+
+The opposite of the CIA triad is sometimes known as the DAD triad:
+
+- If confidentiality is affected, **d**isclosure occurs.
+- If integrity is affected, **a**lteration occurs.
+- If availability is affected, **d**enial occurs.
+
+Here is a fun, alternative look at the
+[CIA and DAD triads, explained with Lord of the Rings references].
+
+## STRIDE
+
+A model of threats developed by Microsoft. It’s a mnemonic for splitting
+security threats into six categories.
+
+It's a helpful tool to think about different potential attacks or
+vulnerabilities.
+
+
+|Threat|Security Control|
+|-|-|
+|**S**poofing|Authenticity|
+|**T**ampering|Integrity|
+|**R**epudiation|Non-repudiability|
+|**I**nformation Disclosure|Confidentiality|
+|**D**enial of Service|Availability|
+|**E**levation of Privilege|Authorisation|
+
+### Spoofing
+It occurs where malicious actors use another person's user credentials without
+their knowledge.
+
+It also includes attacks where other systems mimic or disguise as others, such
+as IP address spoofing in DNS amplification attacks.
+
+The desired property is authenticity, meaning an event or action is trustworthy
+and genuine.
+
+### Tampering
+Only authorised users should be able to modify a system or the data it uses.
+
+The desired property is integrity, meaning an event or action is valid and
+correct.
+
+### Repudiation
+The ability to refuse that something is correct.
+
+The desired property is non-repudiation, meaning an action or event is not easy
+to deny by a person or system.
+
+Examples of a non-reputable control are biometric authentication or mutual TLS.
```

Comment:
Would it be worth adding links for these so the user can look for more info if required?

Reply:
I'd probably only link something like wikipedia here for overview. If this isn't helpful, I can get rid of the examples?

Reply:
We should avoid linking to Wikipedia. I was thinking it could be useful to link to reliable sources for more info on biometric auth and mutual TLS but there aren't any suitable ones, we could just leave it as it is.